### DISCUSSION TITLE
Information carte

### DISCUSSION THREAD

**Discussion by rvtt51** [mytourbook/mytourbook#1081] [Ideas]
Bonjour,
Dans l"évolution de l'application, est-ce qu'il serait possible, lorsque qu'on survole le graphique d'un parcours avec la souris, de voir apparaître le temps et/ou la distance en même temps sur ce graphique et aussi sur la carte du parcours ? Merci.
Bien sportivement.
RV(tt)

**Comment by FJBDev** [comment]
Si je comprends bien votre requete, il me semble que c'est deja possible grace au "Tour Tooltip Info" disponible en haut a gauche de la carte 2D comme ci-dessous:

![image](https://user-images.githubusercontent.com/25235665/234874674-864325e0-3093-4bb4-bc48-e2cb490dc431.png)

**Comment by rvtt51** [comment]
Bonjour. Ce n'est pas l'objet de ma demande.. Ce que je souhaiterais voir apparaitre, c'est en même temps que le survol du graphe de dénivelé avec la souris, sur le point blanc et rouge qui se déplace sur la carte, voir en info bulle le temps et la distance. Savoir qu'à tel endroit sur la carte, j'ai fait x kms et x h-mn. J'espère que mon explication est claire ...

**Reply by FJBDev** [reply]
Dans ce cas, avez-vous active l'infobulle de la carte 2D ?
Dans mon exemple ci-dessous, vous pouvez voir le duree et le kilometrage lors du survol du parcours

![image](https://user-images.githubusercontent.com/25235665/234917500-37500c44-7268-4602-8e90-c5168597e4d8.png)

**Comment by rvtt51** [comment]
Merci du retour. Oui, ça ressemble à ce qui pourrait me convenir. Mais je ne trouve pas le menu qui permet d'avoir la même chose que sur limage jointe. Pouvez-vous me guider ? Merci !

**Reply by FJBDev** [reply]
Le menu se trouve dans le menu glissant de la carte 2D, comme illustre ci-dessous
En esperant que ca vous aidera a le trouver

![image](https://user-images.githubusercontent.com/25235665/234972563-ed1a4244-4c97-419e-b440-b57589c0f581.png)

**Comment by rvtt51** [comment]
Bonjour,
La case est bien cochée sur ma fenêtre mais rien ne s'affiche sur la carte !
![carte_MTB](https://user-images.githubusercontent.com/131953438/235206320-185b8b3d-2ecb-451f-b09c-93a01dd26d01.jpg)

**Reply by FJBDev** [reply]
Pouvez-vous essayer de fermer la carte 2D et de la reouvrir ?

**Comment by rvtt51** [comment]
Bonjour,
J'ai bien le nouveau petit menu associé à  la carte 2D. Mais quelque soit le déplacement de la souris depuis le graphe, l'info point ne me renvoie rien. Les valeurs restent à zéro (distance, temps).

**Comment by rvtt51** [comment]
PJ oubliée !! 
![carto_MTB](https://user-images.githubusercontent.com/131953438/235291986-2ff04dd0-9f64-4631-a19c-f509fe6a5a49.jpg)

**Reply by FJBDev** [reply]
Ok, cest deja mieux avec l'info-bulle...
Pouvez vous attacher une capture decran du graphique de parcours pour voir si l'infobulle du graphique se comporte differemment?
Quelle version de MT utilisez vous?

**Comment by rvtt51** [comment]
Bon, on avance !!! L'info bulle côté graphique, est OK : temps + distance. Mais la carte ne suit pas !! Ci-joint la PJ correspondante
![carto_MTB](https://user-images.githubusercontent.com/131953438/235316402-09b1358b-17c2-47a1-b010-d9d4ae079d08.jpg)

**Comment by GurVJ** [comment]
Est-ce que dans "Option des Traces", l'option "Afficher le parcours survolé/sélectionné" est activée ?
 
![Capture 001_20230429](https://user-images.githubusercontent.com/64196685/235317721-ad04063a-a29c-45df-9cee-4b82330ccf36.jpg)

**Comment by rvtt51** [comment]
Bingo ! La case n'était pas cochée. Je viens de le faire. Maintenant je peux suivre le parcours sur la carte (et non plus par le graphe) et j'ai bien  les infos (personnalisées) : temps et distance.
Merci beaucoup pour votre aide. Le prg est tellement complet qu'on passe à côté de fonctionnalités.  J'ai donc bien la solution à ma demande initiale. Je vais cloturer le post en validant ma réponse.
